app/services/cards_service.rb

```diff
@@ -0,0 +1,38 @@
+class CardsService
+  class << self
+    def parsed_html?(user_id, block_id, url, css_original, css_translated)
+      parsed = false
+
+      if block_id && user_id
+        html = Nokogiri::HTML(open(url))
+
+        original_texts = html.css(css_original).map { |original| original.content }
+        translated_texts = html.css(css_translated).map { |translated| translated.content }
```

Comment:
Line is too long. [91/80]<br>Pass `&:content` as an argument to `map` instead of a block.